Baymax/Logger/LogDetailViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  LogDetailViewController.swift
+//  Baymax
+//
+//  Created by Simon Mitchell on 01/04/2020.
+//  Copyright © 2020 3 SIDED CUBE. All rights reserved.
+//
+
+import UIKit
+
+class LogDetailViewController: UIViewController {
+    
+    let file: LogFile?
+    
+    init(logfile: LogFile) {
+        file = logfile
+        super.init(nibName: nil, bundle: nil)
+        title = file?.url.lastPathComponent
+    }
+    
+    required init?(coder: NSCoder) {
```

Comment:
Does it make sense to allow this to be created via storyboard/nib. Prop personally prefer making `LogFile` non-nullable but it's really no biggy (and just a preference)

Reply:
Unfortunately you have to provide `init?(coder: NSCoder)` if you provide a custom init on `UIViewController` as it's a designated initialiser! I'd also much rather have `LogFile` non-nullable... I'll see what I can do!

Reply:
Not sure if there's anything I can do here other than providing a default `LogFile` in the `init?(coder: NSCoder)` initialiser, but I'd rather keep it optional tbh: a dummy file will exhibit exactly the same behaviour as the file won't be able to be read from disk! If this was a public/open class I'd be a bit more cautious, but because it's internal I think this is okay to leave as-is!

Reply:
No need to block PR approval for this one